.github/workflows/stackhpc.yml

```
@@ -5,7 +5,17 @@ on:
   push:
     branches:
       - main
+    paths-ignore:
+      - dev/**
```

I don't think this is correct as dev/setup-env.sh it is used in CI, deliberately, so we test setting up the cluster. See https://github.com/stackhpc/ansible-slurm-appliance/blob/main/.github/workflows/stackhpc.yml#L47

---

Nice catch, thanks

---

@bertiethorpe Would you mind updating this to use something like:
```suggestion
     paths:
       - !dev/**
       - dev/setup-env.sh
```
to capture this requirement correctly? Relevant docs are [here](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore) and a useful example [here](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#example-including-and-excluding-paths).